### PR TITLE
Search View & Post Update

### PR DESCRIPTION
**Pull request by noginger13**
Updates in prep of glorious data.

### CODE REVIEW

File: client/src/components/add-trip-view/add-trip-view.jsx [hunk 1]
```diff
@@ -81,17 +81,20 @@ function AddTripView() {
...
-      postedOn: new Date(),
-      from: directionData.startAddress,
-      to: directionData.endAddress,
-      seats: seats,
-      dateTime: value,
-      tripDetails: tripValue,
-      cost: directionData.cost,
-      miles: directionData.miles,
-      milesReadable: directionData.milesReadable,
+      title: title,
```
Comment (cello-frodo) [inline client/src/components/add-trip-view/add-trip-view.jsx:84]: Needs a trip id to be posted so that the trip can be queried

**Review by galopyz — APPROVED** [review]
LGT!M

File: client/src/components/add-trip-view/add-trip-view.jsx [hunk 2]
```diff
@@ -81,17 +81,20 @@ function AddTripView() {
...
+      departure: directionData.startAddress,
+      depart_coord: startPos,
+      destination: directionData.endAddress,
+      dest_coord: destPos,
+      distance: directionData.miles,
+      distance_str: directionData.milesReadable,
+      passenger_capacity: seats,
+      price: directionData.cost,
+      duration: directionData.timeReadable,
       seconds: directionData.seconds,
```
Comment (cello-frodo) [inline client/src/components/add-trip-view/add-trip-view.jsx:97]: Needs an empty passenger array set to passengers

**Review by cello-frodo — CHANGES_REQUESTED** [review]
Needs a few parameters added to post the trip. see comments for details.

(passengers and id both need setting)

**Pull request merged**